src/gc/bits.d

```diff
@@ -33,29 +34,37 @@ struct GCBits
     enum BITS_MASK = (BITS_PER_WORD - 1);
     enum BITS_0 = cast(wordtype)0;
     enum BITS_1 = cast(wordtype)1;
+    bool mmap;
```

Comment:
Please move this down a couple of lines so that it is with the variables `data` and `nbits`

Reply:
That also works.